src/components/Icons/Twemoji/Camera.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+
+const SvgTwemoji1F4F7 = props => (
+  <svg viewBox="0 0 47.5 47.5" {...props}>
+    <defs>
+      <clipPath id="Twemoji_1f4f7_svg__a">
+        <path d="M0 38h38V0H0v38z" />
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 16 spaces but found 8. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)